pypeit/spectrographs/gemini_gmos.py

```diff
@@ -116,12 +120,58 @@ def default_pypeit_par():
         par['fluxcalib'] = pypeitpar.FluxCalibrationPar()
         # Always correct for flexure, starting with default parameters
         par['flexure'] = pypeitpar.FlexurePar()
+        # Always correct for flexure, starting with default parameters
+        par['flexure']['method'] = 'boxcar'
 
         # Set the default exposure time ranges for the frame typing
         #par['scienceframe']['exprng'] = [30, None]
 
         return par
 
+    def config_specific_par(self, scifile, inp_par=None):
+        """
+        Modify the PypeIt parameters to hard-wired values used for
+        specific instrument configurations.
+
+        .. todo::
+            Document the changes made!
+
+        Args:
+            scifile (str):
+                File to use when determining the configuration and how
+                to adjust the input parameters.
+            inp_par (:class:`pypeit.par.parset.ParSet`, optional):
+                Parameter set used for the full run of PypeIt.  If None,
+                use :func:`default_pypeit_par`.
+
+        Returns:
+            :class:`pypeit.par.parset.ParSet`: The PypeIt paramter set
+            adjusted for configuration specific parameter values.
+        """
+        par = self.default_pypeit_par() if inp_par is None else inp_par
+        # TODO: Should we allow the user to override these?
+        if self.get_meta_value(scifile, 'dispname')[0:4] == 'R400':
```

Comment:
I think we should allow users to override these. There are some arcs that only readout the centeral chips (science images always readout the full map). For these kind of data I have to use the science image as the arc and tilt if they are red enough.